client/components/course/Settings/index.vue

```diff
@@ -50,25 +59,33 @@
 <script>
 import api from '../../../api/course';
```

Comment:
If you're only using `getContentInventory` from  `api` component, you can destructure it.